src/stories/components/IconSimpleView.jsx

```diff
@@ -8,16 +7,21 @@ import {
 import { Icon } from '../../components'
 import PropTypes from 'prop-types'
 import React from 'react'
+import { message } from 'antd'
 import { useShareIconName } from '../hooks'
 
 const IconSimpleView = (props) => {
-  const { name, showInActionsCopied, iconActions } = props
+  const { name, showInActionsCopied, size, fill } = props
+  //  [COMPUTED_PROPERTIES]
+  let sizeProp = size ? `size={${size}}` : ''
+  let fillProp = fill ? `fill={${fill}}` : ''
+
   // [ADDITIONAL_HOOKS]
   const shareIconName = useShareIconName()
 
   // [HELPER_FUNCTIONS]
   const handleCopyIcon = (e) => {
-    const iconComponent = `<${name} />`
+    const iconComponent = `<${name} ${sizeProp} ${fillProp} />`
```

Comment:
move whitespaces to constants
<img width="401" alt="image" src="https://user-images.githubusercontent.com/25725369/186166056-40912013-7506-488d-a2f7-2cb6fce5375d.png">